src/jbuild

```diff
@@ -1,8 +1,12 @@
 (jbuild_version 1)
 
-(executable
- ((name       wsproxy)
-  (public_name wsproxy)
-  (libraries   (re.str lwt lwt.unix))
+(executables
+ ((names (wsproxy wsproxy_tests))
```

Comment:
One of the nice features of jbuilder is that it helped packages to move modules around to separate modules, submodules and tests. I think we should follow the trend and move the tests to a `tests` folder in line with the rest of the jbuilder packages.

For this specific test it should be enough to add a `library` section to this `jbuild` file exporting `base64`. Then it should be importable in the test as `Wsproxy.Base64`.